src/main/java/com/cisco/trex/stateful/model/stats/PortStatsData.java

```diff
@@ -0,0 +1,182 @@
+package com.cisco.trex.stateful.model.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PortStatsData {
+
+    @JsonProperty("ignore_bytes")
+    public Integer ignore_bytes;
+
+    @JsonProperty("ipv6_n_solic")
+    public Integer ipv6_n_solic;
+
+    @JsonProperty("m_jitter")
+    public Integer m_jitter;
+
+    @JsonProperty("m_l3_cs_err")
+    public Integer m_l3_cs_err;
+
+    @JsonProperty("m_l4_cs_err")
+    public Integer m_l4_cs_err;
+
+    @JsonProperty("m_length_error")
+    public Integer m_length_error;
+
+    @JsonProperty("m_no_id")
+    public Integer m_no_id;
+
+    @JsonProperty("m_no_ipv4_option")
+    public Integer m_no_ipv4_option;
+
+    @JsonProperty("m_no_magic")
+    public Integer m_no_magic;
+
+    @JsonProperty("m_pkt_ok")
+    public Integer m_pkt_ok;
+
+    @JsonProperty("m_rx_check")
+    public Integer m_rx_check;
+
+    @JsonProperty("m_seq_error")
+    public Integer m_seq_error;
+
+    @JsonProperty("m_tx_pkt_err")
+    public Integer m_tx_pkt_err;
+
+    @JsonProperty("m_tx_pkt_ok")
+    public Integer m_tx_pkt_ok;
+
+    @JsonProperty("m_unsup_prot")
+    public Integer m_unsup_prot;
+
+    @JsonProperty("tx_arp")
+    public Integer tx_arp;
+
+    public Integer getIgnore_bytes() {
```

Comment:
it's better to add @JsonProperty(key) annotation for all set/getXXX if it has corresponding field in json response.

Reply:
ok, I will add it.